spyderlib/spyder.py

```diff
@@ -2660,6 +2677,87 @@ def show_tour(self, index):
         self.tour.set_tour(index, frames, self)
         self.tour.start_tour()
 
+    # ---- Check for Spyder Updates
+    def _check_updates_ready(self, feedback):
+        """Called by WorkerUpdates when ready"""
+        from distutils.version import StrictVersion as Version
+
+        version = __version__
+        latest_release = self.worker_updates.latest_release
+        if DEV is None:
+            # In case someone installs a release candidate in which this might
+            # fail.
+            try:
+                update_available = Version(latest_release) > Version(version)
+            except Exception:
+                update_available = False
+        else:
+            update_available = False
+
+        if update_available:
+            msg = _("Spyder {0} is available.".format(latest_release))

```

Comment:
The github page you mean?

This one: https://github.com/spyder-ide/spyder/releases  ?